packages/aquedux-client/src/utils/index.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+/**
+ * Compose a channel name.
+ *
+ * e.g if the channel name has no id, it returns the name argument.
+ * else if the channel name has an id, it returns the name-id
+ */
+export function composeName(name: string, id?: string): string {
```

Comment:
`channelName` and `channelNameFromAction` would be clearer

Reply:
👍